seedvault/src/main/java/com/solanamobile/seedvault/Wallet.java

```diff
@@ -196,6 +196,7 @@ public static long onImportSeedResult(
      * @return an {@link Intent} suitable for usage with
      *      {@link Activity#startActivityForResult(Intent, int)}
      */
+    @RequiresApi(api = SeedVault.MIN_API_FOR_PRIVILEGED_SEED_VAULT)
```

Comment:
Fake wallet needs a wrapper for Android version check for calling this.

Reply:
Good catch, will add